src/Uno.UI.Tests/Windows_UI_Xaml_Data/BindingTests/Given_Binding.cs

```diff
@@ -131,5 +133,67 @@ public void When_ElementName_In_Template_In_Template()
 
 			Assert.AreEqual(SUT.topLevel.Tag, tb.Text);
 		}
+
+		[TestMethod]
+		public void When_Xaml_Object_With_Common_Properties()
+		{
+			var SUT = new Binding_Xaml_Object_With_Common_Properties();
+
+			SUT.ForceLoaded();
+
+			SUT.topLevel.Should().NotBeNull();
+			SUT.topLevel.Text.Should().Be("42");
+
+			var bindingExpression = SUT.topLevel.GetBindingExpression(TextBlock.TextProperty);
+			bindingExpression.Should().NotBeNull();
+
+			var binding = bindingExpression.ParentBinding;
+			binding.Should().NotBeNull();
+			binding.Path.Should().NotBeNull();
+			binding.Path.Path.Should().Be("Tag");
+			// It looks like a bug
+			//binding.ElementName.Should().BeOfType<ElementNameSubject>().Which.Name.Should().Be("topLevel");
```

Comment:
It looks like a bug. Should I fix this as part of this PR (although obviously this doesn't explicitly address the issue the PR is fixing)?

Reply:
I'll clarify that `ElementNameSubject.Name` returns `null` because the generated code for initialization uses a constructor without parameters:
```cs
private global::Windows.UI.Xaml.Data.ElementNameSubject _topLevelSubject = new global::Windows.UI.Xaml.Data.ElementNameSubject();
public global::Windows.UI.Xaml.Controls.TextBlock topLevel
{
	get
	{
		return (global::Windows.UI.Xaml.Controls.TextBlock)_topLevelSubject.ElementInstance;
	}
	set
	{
		_topLevelSubject.ElementInstance = value;
	}
}
```

Reply:
That's fine if it's another issue we're opening to track this.

Reply:
I want to clarify what I should do - just leave a comment, as I already did, or open an issue and describe the problem?

Reply:
Perhaps I should remove the comment and this line of code? Or refer to it in the problem, leaving it?

Reply:
You can leave the line, comment the assertion with the new issue created to track it. Thanks!